numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -3827,6 +3827,7 @@ def add_newdoc(place, name, doc):
     --------
     lib.scimath.sqrt
         A version which returns complex numbers when given negative reals.
+        Note: 0.0 and -0.0 are handled differently for complex inputs.
```

Comment:
Not sure this line is needed, I think just having the example below is enough.

Reply:
The example is in the generated docs for `lib.scimath.sqrt` and won't show up on the same page though